pixel-saver@deadalnix.me/buttons.js

```diff
@@ -48,7 +48,11 @@ function createButtons() {
 	let orders = order.replace(/ /g, '').split(':');
 	
 	orders[0] = orders[0].split(',');
-	orders[1] = orders[1].split(',');
+	
+	// Checking the secondary button orders
+	if(typeof orders[1] != 'undefined') orders[1] = orders[1].split(',');
+	// if doesn't exist then use the default orders
+	else orders[1] = orders[0];
```

Comment:
Bigger issue the performance in this the `(x in y)` :wink: also the orders[1] should have an empty array istead of an empty string. (because you split it by comma for later code)